metadata/src/main/java/org/apache/kafka/controller/ReplicationControlManager.java

```diff
@@ -1419,6 +1421,15 @@ ControllerResult<BrokerHeartbeatReply> processBrokerHeartbeat(
         return ControllerResult.of(records, reply);
     }
 
+    void processExpiredBrokerHeartbeat(BrokerHeartbeatRequestData request) {
+        int brokerId = request.brokerId();
+        clusterControl.checkBrokerEpoch(brokerId, request.brokerEpoch());
+        clusterControl.heartbeatManager().touch(brokerId,
+                clusterControl.brokerRegistrations().get(brokerId).fenced(),
+                request.currentMetadataOffset());
+        log.error("processExpiredBrokerHeartbeat: timed out heartbeat from broker {}.", brokerId);
```

Comment:
What is the operator meant to do when they see this error message? Perhaps we can explain the implications in the log message?

Reply:
I changed it to "controller event queue overloaded. timed out heartbeat from broker {}"